api/admin.go

```diff
@@ -342,3 +351,80 @@ func (a *API) adminUserDelete(w http.ResponseWriter, r *http.Request) error {
 
 	return sendJSON(w, http.StatusOK, map[string]interface{}{})
 }
+
+func (a *API) adminUserUpdateFactorStatus(w http.ResponseWriter, r *http.Request) error {
+	ctx := r.Context()
+	user := getUser(ctx)
+	instanceID := getInstanceID(ctx)
+
+	params := &AdminUserUpdateFactorStatusParams{}
+	jsonDecoder := json.NewDecoder(r.Body)
+	err := jsonDecoder.Decode(params)
+	if err != nil {
+		return badRequestError("Could not read AdminUpdateFactorStatus params: %v", err)
```

Comment:
Since this message goes out to users, maybe disclose only the information they need to fix it?